ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -417,7 +421,9 @@ def setUp(self):
         self.generate_arp_ping_packet()
 
         if self.reboot_type == 'warm-reboot':
-            self.log("Preboot Oper: %s" % self.preboot_oper)
+            (oper_type, cnt) = self.preboot_oper.split(':') if self.preboot_oper and ':' in self.preboot_oper else (self.preboot_oper, 1)
```

Comment:
> (oper_type, cnt) = self.preboot_oper.split(':') if self.preboot_oper and ':' in self.preboot_oper else (self.preboot_oper, 1) [](start = 11, length = 126)

Use if-else block instead of one liner.

Reply:
done